CalPatRec/src/CalHelixFinder_module.cc

```diff
@@ -507,16 +516,22 @@ namespace mu2e {
 
     // double     z_start(0);
     HelSeed._hhits.setParent(_chcol->parent());
+
+    float eDepSum(0.0);
+    size_t nComboHits(0);
+
     for (int i=0; i<nhits; ++i){
       unsigned        hitId   = HfResult._goodhits[i];
       ComboHit*       hit     = &HfResult._chHitsToProcess[hitId];//panelz->_chHitsToProcess.at(hitInfo->panelHitIndex);
 
       ComboHit                hhit(*hit);
       //      hhit._hphi = shphi;
       // hhit._flag.merge(StrawHitFlag::resolvedphi);
-
+      eDepSum = eDepSum + hhit.energyDep();
```

Comment:
Same comment about the EDep average calculation.  The right answer is to implement the average EDep function correctly in ComboHitCollection, and have clients call that.  This needs to be part of this PR.